customcontext/context.h

```diff
@@ -59,34 +59,51 @@
 #include "threaduploadtexture.h"
 #endif
 
+#if QT_VERSION >= 0x050800
+#include <private/qsgdefaultcontext_p.h>
+#include <private/qsgdefaultrendercontext_p.h>
+#endif
 
-#if QT_VERSION >= 0x050200
 struct QSGMaterialType;
+
+#define CONTEXT_CLASS Context
+#define RENDER_CONTEXT_CLASS RenderContext
```

Comment:
With <5.2 support removed, these don't have anymore variation. Could remove the definitions and instead just use target names in the code?

Reply:
Fixed with https://github.com/sailfishos/qtscenegraph-adaptation/pull/1/commits/a70e9a9e883cd0e6e989e18462b33d356ad395ce